components/support/nimbus-cli/src/cmd.rs

```diff
@@ -338,49 +379,50 @@ impl LaunchableApp {
         reset_db: bool,
         json: Option<&Value>,
         log_state: bool,
+        deeplink: Option<&String>,
     ) -> Result<Command> {
         if let Self::Android {
             package_name,
             activity_name,
             ..
         } = self
         {
-            let reset_db_arg = if reset_db {
-                Some("--ez reset-db true".to_string())
+            let mut args: Vec<String> = Vec::new();
+
+            if let Some(deeplink) = deeplink {
+                args.extend(vec![
+                    "-a android.intent.action.VIEW".to_string(),
+                    "-c android.intent.category.DEFAULT".to_string(),
+                    "-c android.intent.category.BROWSABLE".to_string(),
+                    format!("-d {}", deeplink),
+                ]);
             } else {
-                None
-            };
-            let experiments_arg = if let Some(s) = json {
+                args.push(format!("-n {}/{}", package_name, activity_name));
+                args.extend(vec![
+                    "-a android.intent.action.MAIN".to_string(),
+                    "-c android.intent.category.LAUNCHER".to_string(),
+                ]);
```

Comment:
Is there a particular reason for not including the contents of the `push` in the `extend`'s vector?

Reply:
No, you're right! I also changed to using a slice rather than a `vec![]`.